DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/Index.cshtml

```diff
@@ -39,6 +38,17 @@
     </div>
     <partial name="_ActivityTable.cshtml" model="@Model.UsageStatsTableViewModel" />
 
+    <hr />
+    <h2 class="nhsuk-heading-m">Evaluation summary</h2>
+

```

Comment:
This is extra, but I wonder if we want to add a paragraph here explaining what this section covers - worth asking HEE at Show & Tell tomorrow, then they can provide a copy if they think it'd be useful :)